ansible/playbook/roles/install_jenkins/tasks/main.yml

```diff
@@ -8,7 +8,8 @@
     echo "  autoProvisionEnabled: false" >> {{ config_file }}
 
 - name: Restart master
-  command: systemctl restart origin-master-api origin-master-controllers
+  #systemctl restart origin-master-api origin-master-controllers
+  command: systemctl restart openshift.service
```

Comment:
The change to this file is no longer needed since the master branch already handles the presence of `openshift.servive` automatically

Reply:
Really cool what you did on master branch. How can remove this change ?

Reply:
I will attempt to rebase this PR onto the master branch. If that works out, I will force push it here and we should be able to merge without problems :)